__tests__/app.test.js

```diff
@@ -48,3 +48,44 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("return status 200 when successful", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200);
+  });
+  test("return an object with the expected values", () => {
```

Comment:
You should make a specific test to check if the object you are getting is for the article you are requesting. 
i.e. that you are definitely getting article 1 when you request it

Reply:
Thank you for your feedback. I will update my test suite now.